ififuncs.py

```diff
@@ -728,13 +728,13 @@ def get_user():
     Asks user who they are. Returns a string with their name
     '''
     user = ''
-    if user not in ('1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11'):
+    if user not in ('1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'):
         user = input(
-            '\n\n**** Who are you?\nPress 1,2,3,4,5,6,7,8,9,10,11\n\n1. Aoife Fitzmaurice\n2. Gavin Martin\n3. Kieran O\'Leary\n4. Raelene Casey\n5. Noelia Romero\n6. Ana Truchanova\n7. Eoin O\'Donohoe\n8. Yujing Huang\n9. Colm Connolly\n10. Yazhou He\n11. Aoife McGrath\n'
+            '\n\n**** Who are you?\nPress 1,2,3,4,5,6,7,8,9,10,11,12\n\n 1. Aoife Fitzmaurice\n 2. Gavin Martin\n 3. Kieran O\'Leary\n 4. Raelene Casey\n 5. Noelia Romero\n 6. Ana Truchanova\n 7. Eoin O\'Donohoe\n 8. Yujing Huang\n 9. Colm Connolly\n10. Raven Cooke\n11. Yazhou He\n12. Aoife McGrath\n'
         )
-        while user not in ('1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11'):
+        while user not in ('1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'):
             user = input(
-                '\n\n**** Who are you?\nPress 1,2,3,4,5,6,7,8,9,10,11\n\n1. Aoife Fitzmaurice\n2. Gavin Martin\n3. Kieran O\'Leary\n4. Raelene Casey\n5. Noelia Romero\n6. Ana Truchanova\n7. Eoin O\'Donohoe\n8. Yujing Huang\n9. Colm Connolly\n10. Yazhou He\n11. Aoife McGrath\n'
+                '\n\n**** Who are you?\nPress 1,2,3,4,5,6,7,8,9,10,11,12\n\n1. Aoife Fitzmaurice\n2. Gavin Martin\n3. Kieran O\'Leary\n4. Raelene Casey\n5. Noelia Romero\n6. Ana Truchanova\n7. Eoin O\'Donohoe\n8. Yujing Huang\n9. Colm Connolly\n10. Raven Cooke\n11. Yazhou He\n12. Aoife McGrath\'
```

Comment:
this is inconsistent in the opposite way. New entries have an added space and the existing ones do not.

Reply:
Hi oh yes this one should also add the space there.

Reply:
This is why this isn't probably the best decision, as it makes it a bit more complex to maintain - you have to maintain this rule in multiple areas of code, especially if applying the rule elsewhere. It might be easier to just stick with consistent newlines over the aesthetic benefits? Your call ultimately.

Reply:
Also, it's best for commits to do one thing. For example, this commit added new names but it also changed the code and visual style, so it's hard to track the changes in the commit log. These are all things I learned the hard way - btw you an see the commit style I used here: https://github.com/kieranjol/IFIscripts/commits/master

Reply:
yes exactly, I'm also thinking about using \t instead of a space there

Reply:
Tab could be interesting, I never used them. Would that maintain the alignment better?

Reply:
Yes I hope so. Once we have number more than ten, it means the name will be going right for one character. Tab could make it better I think

Reply:
I will have a try for the commit style :D

Reply:
It helps when using git on the command line. I used to always do `git commit -m $name_of_script - $what_the_commit_did %scriptname`, so like `git commit -m "ififuncs - add new staff" ififuncs.py` and then git push if I was happy and did my testing. You might be able to replicate that style when using the website but I'm not sure if it's possible.